renv.lock

```diff
@@ -9,7 +9,7 @@
     ]
```

Comment:
I would change the repo [here ](https://github.com/Biogen-Inc/tidyCDISC/blob/cd08ceb68e9aca1a3da7ad893057e4d3d3834240/renv.lock#L7)to point at a snapshot for stability similar to what was [done in `riskassessment`](https://github.com/pharmaR/riskassessment/blob/8563df1717e94676db59313b3b93b67809ac5c74/renv.lock#L7).

Reply:
done!